CONTRIBUTING.md

```diff
@@ -32,6 +32,19 @@ Your pull request should:
 * Follow the code conventions descriped in [Coding guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)
 * To avoid line ending issues, set `autocrlf = input` and `whitespace = cr-at-eol` in your git configuration
 
+## Contributing `lib.d.ts` fixes
+ 
+The library sources are in: [src\lib\](https://github.com/Microsoft/TypeScript/tree/master/src/lib)

```

Comment:
The trailing slash is disabling the link. I would get rid of the trailing slash or switch to forward slash.